tests/backup_helper.go

```diff
@@ -4920,6 +4948,41 @@ func CreateRuleForBackupWithMultipleApplications(orgID string, appList []string,
 	return preRuleName, postRuleName, nil
 }
 
+type VMRuleType string
+
+const (
```

Comment:
You'll need anyways both for any given Vm right why do you want to specific VMRuleType in general?

Reply:
No, we need not have both for a VM. In my case I am creating only freeze rule and checking how backup works when I give only freeze rule without unfreeze.
If anybody needs a function with both freeze and unfreeze, then they can write a wrapper function which calls
```
err = CreateRuleForVMBackup(freezeRuleName, allVMs, Freeze, ctx)
err = CreateRuleForVMBackup(freezeRuleName, allVMs, Unfreeze, ctx)
```

Reply:
Got it.